### PR TITLE
Add IWA part to the main readme of the repo

### PR DESCRIPTION
**Pull request by sonkkeli**
Also updated the table of contents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,17 @@ draft](https://tools.ietf.org/html/draft-yasskin-wpack-use-cases)).
 
 <!-- TOC -->
 
-- [Explainers](#explainers)
```
Comment (cmfcmf) [inline README.md:16]: IMHO the table of contents does not need to have a reference to the README's title.

**Review by cmfcmf — APPROVED** [review]
LGTM % the addition of the README's title to the TOC.

File: README.md (hunk 1 above)
Comment (sonkkeli) [reply README.md:16]: This got added automatically on save. Should I still remove it?

Comment (cmfcmf) [reply README.md:16]: Hmm, interesting, I would still remove it.

**Pull request merged**